projects/batfish/test/org/batfish/main/BatfishTest.java

```diff
@@ -0,0 +1,68 @@
+package org.batfish.main;
+
+import org.batfish.common.BatfishException;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+
+public class BatfishTest {
+
+    @Test(expected = BatfishException.class)
+    public void testNotExistingPath() {
+        List<Path> result = Batfish.listAllFiles(Paths.get("./test/notExist"));
```

Comment:
Should test the contents of the exception message. I believe you can use [`ExpectedException`](http://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html) for this.